src/Altinn.AccessManagement/Program.cs

```diff
@@ -269,6 +272,11 @@ void ConfigureServices(IServiceCollection services, IConfiguration config)
                 options.RequireHttpsMetadata = false;
             }
         });
+    }
+    else
+    {
+        logger.LogError("Unable to setup authentication. Missing altinn OidcProvider config.");
```

Comment:
@TheTechArch What is actually best for deploy? Handling and logging missing config or fail completely?

Reply:
Good question. Failing will cause the old pod to still run and would not introduce downtime